docs/examples/alert/description.vue

```diff
@@ -1,7 +1,10 @@
 <template>
-  <el-alert
-    title="with description"
-    type="success"
-    description="This is a description."
-  />
+  <div style="max-width: 600px">
+    <el-alert
+      title="with description"
+      type="success"
+      description="This is a description."
+    />
+  </div>
 </template>
+<script setup lang="ts"></script>
```

Comment:
Is this script tag without content necessary?

Reply:
Oh, it's added automatically when saving the file.

Reply:
Is this automatically added by the plugin you installed? Perhaps we can remove it?

Reply:
Let me try.